MaterialMonitor/ConfigurationWindow.xaml

```diff
@@ -12,6 +12,31 @@
     <DockPanel LastChildFill="True" Background="#FFE5E5E5" Margin="0,5">
         <TextBlock DockPanel.Dock="Top" TextWrapping="Wrap" Margin="5,0" Text="{x:Static resx:MaterialMonitor.p1}"/>
         <TextBlock DockPanel.Dock="Top" TextWrapping="Wrap" Margin ="5" Text="{x:Static resx:MaterialMonitor.p2}"/>
+        <Grid DockPanel.Dock="Top" Margin="0,5" >
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="250" />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="Auto" />
+            </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+            <Label x:Name ="maxStationDistanceLabel" Grid.Row="0" Grid.Column="0" Margin="0,5" VerticalContentAlignment="Center" Content="{x:Static resx:MaterialMonitor.tab_station_distance}"/>
+            <TextBox x:Name="maxStationDistanceInt" Grid.Row="0" Grid.Column="1" Margin="0,5"
+                     PreviewTextInput="EnsureValidInteger"
+                     KeyDown="maxStationDistance_KeyDown"
+                     LostFocus="maxStationDistance_LostFocus"
+                     TextAlignment="Center"
+                     VerticalContentAlignment="Center" Height="25" Width="60"/>
+            <TextBlock Grid.Row="1" Grid.Column="2" Margin="20" TextWrapping="Wrap" Text="{x:Static resx:MaterialMonitor.p3}" VerticalAlignment="Center"/>
```

Comment:
Playing with this a bit, I like a margin of `Margin="10, 0"`. Thoughts?

Reply:
Should we move this configuration option down below to where the buttons are?
Should there be a vocal response when the clipboard is set?

Reply:
I don't feel strongly either way. I moved it and it looks fine.

Reply:
I added notification to the `Route details` script for the clipboard when Voice Attack is not active.